app/services/error-handler.js

```diff
@@ -0,0 +1,36 @@
+import Service, { inject as service } from '@ember/service';
+
+export default class ErrorHandlerService extends Service {
+
+@service notify;
+
+@service l10n;
+
+handleError(error) {
+  const errorStatus =  (error?.errors?.[0]?.status) ? parseInt(error?.errors?.[0]?.status) : error;
+  switch (errorStatus) {
+    case 403:
+      this.notify.error(this.l10n.t('You are not authorized to access the page.'));
+      break;
+    case 422:
+      this.notify.error(this.l10n.t('Some required field is missing or is in wrong format.'));
+      break;
+    case 429:
+      this.notify.error(this.l10n.t('There are too many requests. Please try after sometime.'));
+      break;
+    case 401:
```

Comment:
Except 401, 403 and 429, remove all, and show `errors[0].detail` or something which returns from the server. More often than not, it has more context than predefined messages